packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts

```diff
@@ -90,16 +94,28 @@ export function getCommonConfig(wco: WebpackConfigOptions): Configuration {
             : [noModuleScript];
         }
 
-        // For differential loading we don't need to generate a seperate polyfill file
+        // For full build differential loading we don't need to generate a seperate polyfill file
         // because they will be loaded exclusivly based on module and nomodule
-        const polyfillsChunkName = buildBrowserFeatures.isDifferentialLoadingNeeded()
-          ? 'polyfills'
-          : 'polyfills-es5';
+        const polyfillsChunkName =
+          fullDifferential && differentialLoadingNeeded ? 'polyfills' : 'polyfills-es5';
 
         entryPoints[polyfillsChunkName] = [path.join(__dirname, '..', 'es5-polyfills.js')];
+        if (!fullDifferential && differentialLoadingNeeded) {
+          // Add zone.js legacy support to the es5 polyfills
+          // This is a noop execution-wise if zone-evergreen is not used.
+          entryPoints[polyfillsChunkName].push('zone.js/dist/zone-legacy');
+        }
         if (!buildOptions.aot) {
+          // If not performing a full differential build the JIT polyfills need to be added to ES5
+          if (!fullDifferential && differentialLoadingNeeded) {
+            entryPoints[polyfillsChunkName].push(path.join(__dirname, '..', 'jit-polyfills.js'));
```

Comment:
`jit-polyfills.js` and `buildOptions.polyfills` are already added to the `polyfills` chunk.

 This would causes having polyfills to be ion both `polyfills-es5,js` and `polyfills.js`

https://github.com/angular/angular-cli/blob/0ea63bf3d2dae95ff9b0d35db42694c1a0ad4f4c/packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts#L122-L136